poverty/src/poverty/County.java

```diff
@@ -0,0 +1,19 @@
+package poverty;
+
+public class County {
+	
+	public String name;
+	public String pov_percent;
+	public String pov_count;
+	public String med_income;
+
+	public County(String line) {
+		this.name = line.substring(193, 238);
```

Comment:
Nitpick - since the file format could at some point change, might recommend setting these ints as constants which would map to the type of data they are associated with